app/src/ui.cpp

```diff
@@ -79,6 +80,8 @@ void UI::showMainWindow()
     // view
     m_view->rootContext()->setContextProperty("daemon", m_daemon);
     m_view->rootContext()->setContextProperty("ui", this);
+    m_view->rootContext()->setContextProperty("_keyboardLayout", m_keyboardLayoutProvider);
```

Comment:
Why use an underscore? 
```suggestion
    m_view->rootContext()->setContextProperty("keyboardLayout", m_keyboardLayoutProvider);
```